### PR TITLE
Update Safari iOS versions

### PR DESCRIPTION
**Pull request by a2sheppy**
Added Safari iOS versions 11.2, 11.3, and 11.4.

### CODE REVIEW

File: api/RTCIceTransport.json [hunk 1]
```diff
@@ -5,43 +5,43 @@
...
-            "version_added": null
+            "version_added": "15"
           },
           "edge_mobile": {
-            "version_added": null
+            "version_added": false
           },
           "firefox": {
-            "version_added": null
+            "version_added": false
```
Comment (caugner) [inline api/RTCIceTransport.json]: The title of the PR implies that only Safari iOS versions are changed. However, you have also set other browsers' `version_added` to `false`. This should be incorporated into the PR title.

Comment (caugner) [reply api/RTCIceTransport.json]: Ah, I just saw that the changes are also in #2461, so there probably in here by accident. You might want to remove the commit from the `update-safari-ios-versions` branch.

**Comment by connorshea** [comment]
I'm fairly certain they didn't release new versions of Safari for iOS 11.2 or 11.4.

Safari 11.1 was released for iOS 11.3: https://webkit.org/blog/8216/new-webkit-features-in-safari-11-1/

Otherwise, any changes were performance/security related and didn't introduce any new features.

**Comment by a2sheppy** [comment]
Actually, in that case, this entire PR is invalid.

**Comment by connorshea** [comment]
We might want to write some docs with guidelines for Safari versions so people don't get confused, or change it to match the iOS version? I'm not sure.

**Pull request closed without merging**